### DISCUSSION TITLE
Poetry install in an environment.yml?

### DISCUSSION THREAD

**Discussion by austinmw** [python-poetry/poetry#7614] [Q&A]
Hi, is it possible to place `poetry install` of a local package inside an environment.yml?

I'd like to create a new conda environment and then also install a local package that includes a `pyproject.toml` with poetry. I can do this in two steps, but can it be done in one?

**Comment by ralbertazzi** [comment]
No, and I don't expect this to ever get supported by Poetry. The entire Python community is building around pyproject.toml, while Conda serves a different use case. If you really want to run a single command then you should move everything inside your environment.yml and transform this into a Conda issue
